File: vehicle.py
from rental_item import RentalItem

class Vehicle(RentalItem):
    def __init__(self, id, name, brand, model):
        super().__init__(id, name)
        self.brand = brand
        self.model = model

    def get_brand(self):
        return self.brand

    def get_model(self):
        return self.model

File: Vehicle.java
public class Vehicle extends RentalItem {
    private String brand;
    private String model;

    public Vehicle(String id, String name, String brand, String model) {
        super(id, name);
        this.brand = brand;
        this.model = model;
    }

    public String getBrand() {
        return brand;
    }

    public String getModel() {
        return model;
    }
}
